### DISCUSSION TITLE
Object Click Handling

### DISCUSSION THREAD

**Discussion by dr14nium** [sceneview/sceneview#544] [Q&A]
I worked with the Geospatial API to present 3D land and spatial data. I ignin when the 3D object is clicked, it will popup information about the object. Is this possible with SceneView?

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, use the `onTap` parameter on any node:

```kotlin
SceneView {
    ModelNode(
        modelInstance = model,
        onTap = { motionEvent, node ->
            // This node was tapped
        }
    )
}
```

Works on `ModelNode`, `CubeNode`, `SphereNode`, and all other node types. In AR scenes, the same pattern works inside `AnchorNode`.
